### DISCUSSION TITLE
Welcome to mwptools Discussions!

### DISCUSSION THREAD

**Discussion by stronnag** [stronnag/mwptools#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by larrya18** [comment]


**Reply by stronnag** [reply]
Please generate separate issues rather than spamming the discussion announcement. Reading the User Guide may also help.
